### ISSUE TITLE
Player not looping when playing HLS on Chrome

### ISSUE THREAD

**Issue by moret** [clappr/clappr#548]
Hello folks.

When creating the `Player` with `loop: true` the video does not loop on Chrome, but loops on Safari.

Tested with:
````html
<html>
<head>
  <script type='text/javascript' src='http://cdn.clappr.io/latest/clappr.min.js'></script>
</head>
<body>
  <div id='player'></div>
<script>
var player = new Clappr.Player({
  autoPlay: true,
  loop: true,
  parentId: '#player',
  source: 'http://cdn-fms.rbs.com.br/vod/hls_sample1_manifest.m3u8'
});
</script>
</body>
</html>
````

**Comment by towerz** [comment]
This feature was not implemented for the HLS playback, but I think it's simple enough to implement.

**Comment by moret** [comment]
Confirmed, fixed on `0.2.12`.
